ArduCopter/mode_throw.cpp

```diff
@@ -283,7 +286,7 @@ bool ModeThrow::throw_detected()
 
     // Once a possible throw condition has been detected, we check for 2.5 m/s of downwards velocity change in less than 0.5 seconds to confirm
     bool throw_condition_confirmed = ((AP_HAL::millis() - free_fall_start_ms < 500) && ((inertial_nav.get_velocity_z_up_cms() - free_fall_start_velz) < -250.0f));
-
+    
```

Comment:
accidental white space change.

Reply:
Thanks a lot for taking a look, what other altitude would you recommend? 
I used EKF origin before as the location of AHRS initialization (to my understanding). I suppose I didn't consider a case where the user moves up or down a lot afterwards (mountainous terrain?)